cirq/ops/common_gates.py

```diff
@@ -387,15 +387,20 @@ def controlled(self,
                    control_qid_shape: Optional[Tuple[int, ...]] = None
                   ) -> raw_types.Gate:
         """
-        Specialize controlled for ZPow to return corresponding CZPow when
-        controlled by a single qubit.
+        Specialize controlled for ZPow to return corresponding controlled CZPow
+        when the last control (which acts first semantically) is a default-type
+        control qubit.
         """
         result = super().controlled(num_controls, control_values,
                                     control_qid_shape)
-        if (result.control_values == ((1,),) and  # type: ignore
-                result.control_qid_shape == (2,)):  # type: ignore
+        if (isinstance(result, controlled_gate.ControlledGate) and
+                result.control_values[-1] == (1,) and
```

Comment:
Does this blow up when calling `controlled()` with an empty list of qubits?

Reply:
No, that's why I still kept the super().controlled above. It populates sane defaults for all unspecified args.
Added additional tests to cover this case.